ts/src/krakenfutures.ts

```diff
@@ -1591,6 +1591,90 @@ export default class krakenfutures extends Exchange {
         return this.safeBalance (result);
     }
 
+    async fetchFundingRate (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const fundingRates = await this.fetchFundingRates ([ symbol ], params);
+        for (let i = 0; i < fundingRates.length; i++) {
+            const entry = fundingRates[i];
+            if (entry['symbol'] === market['symbol']) {
+                return entry;
+            }
+        }
```

Comment:
@youknowone `fetchFundingRates` should return the response already filtered by `symbol`, isn't this enough? `const response = this.safeValue(fundingRates,0)` ?

Reply:
Thanks, I updated it. I realized fetchFundingRate in base Exchange has default implementation. Should I remove this one and let it use the default implementation?